Tests/test_PDB.py

```diff
@@ -1205,6 +1205,118 @@ def test_Vector_angles(self):
                         "Want %r and %r to be almost equal" % (axis.get_array(), caxis.get_array()))
 
 
+class StructureAlignTests(unittest.TestCase):
+
+    def test_StructAlign(self):
+        """Tests on module to align two proteins according to a FASTA alignment file."""
+        al_file = "PDB/alignment_file.fa"
+        pdb2 = "PDB/1A8O.pdb"
+        pdb1 = "PDB/2XHE.pdb"
+        with open(al_file, 'r') as handle:
+            records = AlignIO.read(handle, "fasta")
+        p = PDBParser()
+        s1 = p.get_structure('1', pdb1)
+        p = PDBParser()
+        s2 = p.get_structure('2', pdb2)
+        m1 = s1[0]
+        m2 = s2[0]
+        al = StructureAlignment(records, m1, m2)
+        self.assertFalse(al.map12 == al.map21)
+        self.assertTrue(len(al.map12), 566)
+        self.assertTrue(len(al.map21), 70)
+        chain1_A = m1["A"]
+        chain2_A = m2["A"]
+        self.assertEqual(chain1_A[202].get_resname(), 'ILE')
+        self.assertEqual(chain2_A[202].get_resname(), 'LEU')
+        self.assertEqual(chain1_A[291].get_resname(), chain2_A[180].get_resname())
+        self.assertNotEqual(chain1_A[291].get_resname(), chain2_A[181].get_resname())
+
+
+class SuperimposerTests(unittest.TestCase):
+
+    def test_Superimposer(self):
+        """Test on module that superimpose two protein structures."""
+        pdb1 = "PDB/1A8O.pdb"
+        p = PDBParser()
+        s1 = p.get_structure("FIXED", pdb1)
+        fixed = Selection.unfold_entities(s1, "A")
+        s2 = p.get_structure("MOVING", pdb1)
+        moving = Selection.unfold_entities(s2, "A")
+        rot = numpy.identity(3).astype('f')
+        tran = numpy.array((1.0, 2.0, 3.0), 'f')
+        for atom in moving:
+            atom.transform(rot, tran)
+        sup = Superimposer()
+        sup.set_atoms(fixed, moving)
+        self.assertTrue(numpy.array_equal(sup.rotran[0], numpy.array([[1.0000000000000002, -3.8823044778979465e-10, -1.278846473162787e-10], [3.8823072881499776e-10, 1.0, -5.086249268981824e-10], [1.278846473162787e-10, 5.086251524122343e-10, 1.0000000000000002]])))
+        self.assertTrue(numpy.array_equal(sup.rotran[1], numpy.array([-1.0000000256926143, -1.9999999990010409, -2.9999999879063353])))
+        self.assertAlmostEqual(sup.rms, 4.7597824285176668e-07, places=3)
+        atom_list = ['N', 'C', 'C', 'O', 'C', 'C', 'SE', 'C', 'N', 'C', 'C',
```

Comment:
Instead of an array, I could change it into a string e compare them. Even though there might be a difference at the atom level, the overall structure representation should be the same.